torch/ao/quantization/observer.py

```diff
@@ -346,8 +420,8 @@ class MinMaxObserver(_ObserverBase):
         reduce_range: Reduces the range of the quantized data type by 1 bit
         quant_min: Minimum quantization value. If unspecified, it will follow the 8-bit setup.
         quant_max: Maximum quantization value. If unspecified, it will follow the 8-bit setup.
-        memoryless: Boolean that controls whether observer removes old data when a new input is seen.
-                    This is most useful for simulating dynamic quantization, especially during QAT.
+        compute_dtype: if compute_dtype is not set, dtype is the compute_dtype, if compute_dtype
```

Comment:
I think a better way to describe this is: if compute_dtype is None, then both activation_dtype and compute_dtype are dtype, if compute_dtype is not None, then we may have a different compute_dtype than the activation_dtype